module/VuFind/src/VuFind/Role/PermissionProvider/PluginManager.php

```diff
@@ -65,7 +66,8 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         InsecureCookie::class => InsecureCookieFactory::class,
         IpRange::class => IpRangeFactory::class,
         IpRegEx::class => IpRegExFactory::class,
-        Role::class => \Laminas\ServiceManager\Factory\InvokableFactory::class,
+    Role::class => \Laminas\ServiceManager\Factory\InvokableFactory::class,
```

Comment:
There seem to be whitespace problems down here too...

Reply:
thanks @demiankatz I've included this in my follow-up